docs/Code-Standards.md

```diff
@@ -8,61 +8,78 @@ Please familiarise yourself with our:
 
-## Quality Assurance
+- Readability is better than Don't Repeat Yourself (DRY) code. This is especially true in our end-to-end tests where we try to keep the tests (not the logic of how they're run) as simple as possible).
 
-We have code maintainers across our repos ([Simorgh](https://github.com/bbc/simorgh)/[Psammead](https://github.com/bbc/psammead)), who are responsible for ensuring the quality of the code and are a good source for enquiries.
+- Smaller PRs are better, and it's much better for visibility if you open your PRs as draft PRs as early as possible.
 
-#### Issue/PR(Pull Request)
+## Automated Processes
 
-- Each PR should be related to an issue.
-
-- Each issue should be refined by the responsible pod/team before work on a PR is started.
-
-- Each PR needs review by at least one code maintainers and by 2 engineers in total.
-
-#### Unit Tests
-
-- Unit testing is a mixture of Jest snapshot testing and enzyme assertion testing.
-
-- We push automated testing as far down the testing pyramid as possible. This means that if something can be tested via unit tests, to ensure we have a test coverage of > 80% on this repo.
-
-- For strictly **presentational** components Snapshots tests must be present and assertion testing is desirable.
-
-- For **containers** or otherwise logic handling code snapshot testing is not desirable and assertion testing is mandatory.
-
-#### Linting standard
-
-A combination of the [Airbnb Styleguide](https://github.com/airbnb/javascript/tree/master/react) and [Prettier](https://github.com/prettier/prettier) is used as our linting standards.
-
-#### Automated Processes
-
-- Linting for code
+- Linting of our code: A combination of the [Airbnb Styleguide](https://github.com/airbnb/javascript/tree/master/react) and [Prettier](https://github.com/prettier/prettier) is used as our linting standards.
+- Issue and PR templates should be followed, then refined by the relevant internal BBC team. We have no templates for hot-fixes which are what we call live and default branch patches; in such cases maximum speed of work is the priority.
+- Most PRs will require code owner approval. This requirement and its enforcement is entirely automated. We have code maintainers across our repos ([Simorgh](https://github.com/bbc/simorgh)/[Psammead](https://github.com/bbc/psammead)), who are responsible for ensuring the quality of the code and are a good source for enquiries.
 - AMP validation for pages
 - BBC a11y validation for pages
 - Lighthouse validation
 - Code Climate Quality tests
 
-#### Manual Processes
+## Manual Processes
 
+- Open your PRs as early as possible, as draft PRs.
+- Add your issues and PRs to the project "Simorgh".
```

Comment:
Worth mentioning that they should add the relevant label for whichever stream of work it is for?

Reply:
Yes please write that as a suggestion @amywalker1987

Reply:
Done